client/src/cmdlfnexwatch.c

```diff
@@ -24,12 +24,16 @@
 #include "cmdlft55xx.h"    // clone..
 #include "cmdlfem4x05.h"   //
 #include "cliparser.h"
+#include <math.h>
+
 
 typedef enum {
     SCRAMBLE,
     DESCRAMBLE
 } NexWatchScramble_t;
 
+typedef unsigned char BYTE;
```

Comment:
we have spent hours on removing unsigned chars,  and byte references.  
Please use  `uint8_t` instead